pkg/frontend/load.go

```diff
@@ -1511,6 +1527,29 @@ func rowToColumnAndSaveToStorage(handler *WriteBatchHandler, forceConvert bool,
 						cols[i] = d
 					}
 				}
+			case types.T_uuid:
+				cols := vec.Col.([]types.Uuid)
```

Comment:
Use vector.MustTCol or vector.GetFixedValues instead of cast.

Reply:
OK，no problem